### ISSUE TITLE
Mastodome does not support accessibility features

### ISSUE THREAD

**Issue by bobstechsite** [Mastodome/mastodome-qt#86]
* All controls in the GUI should be readable with a screen reader
* The GUI should support the alt text provided with images & video in toots
* CLI should display alt text for images & video in toots
